ext/standard/tests/general_functions/gh10239_1.phpt

```diff
@@ -0,0 +1,18 @@
+--TEST--
+GH-10239 (proc_close after proc_get_status always returns -1)
+--SKIPIF--
+<?php
+if (PHP_OS != "Linux") die("skip, only for linux");
+if (getenv("SKIP_SLOW_TESTS")) die('skip slow test');
+?>
+--FILE--
+<?php
+$p = proc_open('sleep 1', array(), $foo);
+do {
+    usleep(100000);
+    $s = proc_get_status($p);
+} while ($s['running']);
+echo proc_close($p), PHP_EOL;
+?>
+--EXPECT--
+0
```

Comment:
The test description says it should return -1

Reply:
The test description maybe has an unclear title.
I just copied the bug report title: the bug is that it always returns -1 in this case. The correct behaviour is returning 0 like what is tested here :)